### ISSUE TITLE
NestedDotKeys with false and 0 are returned as undefined

### ISSUE THREAD

**Issue by rupal-bq** [mrodrig/json-2-csv#230]
## Background Information
- Module Version: `json-2-csv-cli v4.0.1`
- Node/Browser Version: `v14.20.0 `

The issue I'm reporting is with:
- [x] json2csv
- [ ] csv2json

I have...
- [x] searched to see if an issue has already been reported.
- [x] verified that my JSON/CSV data is valid (using something like http://jsonlint.com or https://csvlint.io/).
- [x] tried upgrading to the latest version of json-2-csv (since the issue may already be fixed).

## Expected Behavior
```
"A\\.B,B\\.C,C\\.D,D\\.E,E\\.F\nfalse,true,1,0,abc"
```

## Actual Behavior
```
"A\\.B,B\\.C,C\\.D,D\\.E,E\\.F\nundefined,true,1,undefined,abc"
```

## Data Sample 

JSON:
```json
[{
	"A.B": false,
	"B.C": true,
	"C.D": 1,
	"D.E": 0,
	"E.F": "abc"
}]
```

## Code Example
Save above sample data to json file and run cli tool
```
json2csv data.json
```

**Comment by mrodrig** [comment]
Hi @rupal-bq, thanks for reporting this. I think I know what might be causing this behavior. I'll take a look and will get back to you with an update. Thanks again!

**Comment by mrodrig** [comment]
Re-opening because the fix is published on `json-2-csv` but a new version of `json-2-csv-cli` still needs to be validated and pushed to NPM.

**Comment by mrodrig** [comment]
Thanks again for reporting this @rupal-bq! I just published `json-2-csv` version `3.20.0` and updated the CLI module (`@mrodrig/json-2-csv-cli`) to version `4.1.0` to fix the issue you reported. It was ultimately a bug in a dependency module I created to spin off some of the reusable logic which wasn't handling these falsy values correctly. Thanks again, and please feel free to reach out if you run into any other behavior that doesn't seem right.

**Comment by rupal-bq** [comment]
@mrodrig  Thanks a lot for the quick response.
